packages/cli/package.json

```diff
@@ -41,16 +41,19 @@
     "node": ">= 14"
   },
   "dependencies": {
-    "@vercel/build-utils": "5.7.5",
-    "@vercel/go": "2.2.24",
-    "@vercel/hydrogen": "0.0.38",
-    "@vercel/next": "3.3.9",
-    "@vercel/node": "2.8.6",
-    "@vercel/python": "3.1.34",
-    "@vercel/redwood": "1.0.45",
-    "@vercel/remix": "1.1.7",
-    "@vercel/ruby": "1.3.50",
-    "@vercel/static-build": "1.1.1"
+    "@vercel/build-utils": "workspace:5.7.5",
+    "@vercel/go": "workspace:2.2.24",
+    "@vercel/hydrogen": "workspace:0.0.38",
+    "@vercel/next": "workspace:3.3.9",
+    "@vercel/node": "workspace:2.8.6",
+    "@vercel/python": "workspace:3.1.34",
+    "@vercel/redwood": "workspace:1.0.45",
+    "@vercel/remix": "workspace:1.1.7",
+    "@vercel/routing-utils": "workspace:2.1.3",
+    "@vercel/ruby": "workspace:1.3.50",
+    "@vercel/static-build": "workspace:1.1.1",
+    "qs": "6.11.0",
+    "update-notifier": "5.1.0"
```

Comment:
```suggestion
```

@Ethan-Arrowood Pretty sure we just removed `update-notifier`. 

cc @cb1kenobi